conduct.html

```diff
@@ -87,6 +87,12 @@ <h2>Scope</h2>
         account, or acting as an appointed representative at an online or
         offline event. Representation of a project may be further defined and
         clarified by project maintainers.
+
+        The project spaces under this Code of Conduct are:
+
+        * Issue trackers of the repositories under The Rails organization at GitHub (https://github.com/rails)
```

Comment:
* should cover source code, wikis, issues, pulls, etc. So probably say "All Rails organization repositories on GitHub" ?